### PR TITLE
Building with Responsive Design

### PR DESCRIPTION
**Pull request by tacodtripe**
We tried to build a clone of "the Next Web magazine" (https://thenextweb.com/). 
The purpose of this exercise is to learn about web responsiveness and its best practices.
In this project, we used many HTML/CSS concepts, like: 
- CSS grid
- CSS flexbox
- Media queries
- Backgrounds

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,686 @@
...
+                <a href="#">About</a>
+                <a href="#">Advertise</a>
+                <a href="#">Jobs</a>
+                <a href="#">Contact</a>
+            </div>
+        </div>
+
+        <a class="navigation-logo" href="#">
+                <img class="top-nav-tnw-logo" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="tnw logo">
+        </a>
```
Comment (majovanilla) [inline index.html:32]: - [ ] Check the size of the logo in the tablet and mobile versions. It's not behaving correctly.
Screenshot from 2020-03-18 16-19-01
Screenshot from 2020-03-18 16-19-16

File: index.html [hunk 2]
```diff
@@ -0,0 +1,686 @@
...
+
+        <div class="latest-news general-container">
+            <h1 class="section-title"><a  class="section-title-a" href="#">Latest News</a></h1>
+            <div class="latest-news-items-container">
+                <div class="latest-news-item">
+                    <img class="latest-news-item-img" src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/03/Untitled-design21-516x270.png" alt="item 1">
+                    <div class="latest-news-item-text-container">
+                        <h3 class="latest-news-item-h3"><a class="latest-news-item-h3-a" href="#">No clue how to handle coronavirus as a startup founder? Here's what we're doing</a></h3>
+                        <h4 class="latest-news-item-h4"><a class="latest-news-item-h4-a" href="#">Patrick de Laive</a> - 4 hours ago</h4>
+                    </div>
```
Comment (majovanilla) [inline index.html:125]: - [ ] Check the size of the articles' images in the mobile version. They should be smaller.

- [x] Check the section's title. It should be bold and larger.

Screenshot from 2020-03-18 16-24-52

Original:
Screenshot from 2020-03-18 16-24-59

- [x] Check the tablet version. The articles should display three in a row. This happens around 990px screen size, so take a look at your media queries px. Yours are like:
Screenshot from 2020-03-18 16-26-06

Should be like:
Screenshot from 2020-03-18 16-26-14

File: index.html [hunk 3]
```diff
@@ -0,0 +1,686 @@
...
+            </div>
+        </div>
+
+        <div class="general-container">
+                <div class="funding-header">
+                    <h1 class="section-title"><a class="section-title-a" href="#"> Latest funding rounds on Index.co ></a></h1>
+                    <div class="funding-navbar">
+                        <a class="funding-navbar-a" href="#">Add info</a>
+                        <a class="funding-navbar-a" href="#">Get updates by email</a>
+                        <a class="funding-navbar-a last-nav-item" href="#">See more</a>
```
Comment (majovanilla) [inline index.html:187]: - [x] This section also displays differently around 990px. It should have 3 elements and occupy 100% width but it's showing this:
Screenshot from 2020-03-18 16-30-00

The original:
Screenshot from 2020-03-18 16-30-00
Screenshot from 2020-03-18 16-30-19

- [x] There's a moment when this section has two articles in one row, that never happens on the original page. Please fix it.
Screenshot from 2020-03-18 16-31-00

- [x] In the mobile and desktop version the image it's too big. Please adjust it.
Screenshot from 2020-03-18 16-32-02

Screenshot from 2020-03-18 16-33-09

File: index.html [hunk 4]
```diff
@@ -0,0 +1,686 @@
...
+                <div class="rubric-thumbnail">
+                    <h2 class= "section-title"><a class="section-title-a" href="#">Apps ></a></h2>
+
+                    <div class="rubric-background-container">
+                        <div class="second-tile-gradient">
+                        </div>
+                        <div class="rubric-thumbnail-head">
+                            <h4 class="most-popular">MOST POPULAR</h4>
+                            <h2><a class="rubric-first-title" href="#">Logitech built a 3-in-1 charger for Apple fans</a></h2>
+                            <h5 class="rubric-second-title"><a class="rubric-second-title" href="#">Napier Lopes</a> - 20 hours ago</h5>
```
Comment (majovanilla) [inline index.html:285]: - [x] All the articles in this section should rearrange in the tablet version to two in a row. Your page shows three at some point. This is also related to the media query, so check your px breaks.

- [x] When changing to the mobile version, your articles are displaying in two columns when they should occupy 100% width. Check the breakpoints to match the original page!

- [x] In the mobile version your titles are not sized appropriately (The main title is too small and the red titles too large).  Check the font size and make sure they are also responsive.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,686 @@
...
+                
+        </div>
+
+        <div class="latest-deals general-container">
+
+                    <div class="latest-deals-title">
+                        <h2 class="section-title"><a class="section-title-a" href="#">Latest Deals ></a></h2>
+                        <h3><a class="funding-navbar-a last-nav-item" href="#">See all Deals</a></h3>
+                    </div>
+
```
Comment (majovanilla) [inline index.html:542]: - [x] This section has larger images in the mobile version too. Once you fix the Latest news, you will fix this one if they have the same classes.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,1117 @@
...
+  width: 4rem;
+}
+
+#heart-icon {
+  color: red;
+}
+
+@media (max-width: 980px) {
+  header {
+    display: grid;
```
Comment (majovanilla) [inline style.css]: - [x] This media query is giving some issues, please check that is the correct `max-width`

File: style.css [hunk 7]
```diff
@@ -0,0 +1,1117 @@
...
+    top: 10%;
+    right: 0%;
+    font-size: 1.5vw;
+  }
+}
+
+@media (max-width: 774px) {
+  footer {
+    height: 50vh;
+  }
```
Comment (majovanilla) [inline style.css]: - [x] Check this one, too!

**Review by majovanilla — CHANGES_REQUESTED** [review]
Hey team!

## You did a *great* job on this project! 💜 
- Good Readme file
- Good Pull Request Message


There are still some changes 👇 you need to implement to approve this request 🔍

### Required Changes: ✍️

- [x] Your project's width it's not the right one. Below 1200px the original page has very small margins, and at some smaller screen sizes, some elements occupy 100% width. Please check this and adjust your page.

- [ ] As you will see in the comments below, the breakpoints you are using are not 100% the same as the original page and when you are close to the break, the page doesn't have the same look as the original page. Please check this out.

- [ ] Take another look at the fonts because in different sections they lose proportion. Make sure they have the same size as the original page and take a closer look at how they change when you change screen sizes.

- [ ] Check the rest of the comments below.


### Optional Changes:

If you want to make your project even better 🔥💥, I have a few suggestions you can implement.  

These are completely [Optional]: 🦔

- Your project would look sooo much better if you include the hover effects on the links of the page, like the original version.

## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes ♻️ 🚥 

Hi **TEAM**, you have really done a great job working on the comments by the previous TSE 😄 We are almost there, let us improve the code by making the project responsive by removing the horizontal scroll on ```small screen```
ffx

- Kindly add a hover effect to the links as in the original

Kindly make these changes and submit another code review 👍 
_Happy coding_ 👨‍💻 
You can find me on slack Daniel Larbi Addo

**Review by addod19 — APPROVED** [review]
## Status: Approved 🏆 🎈 

Hi @tacodtripe  and @CalyCherkaoui, congratulations 👍 on fixing all the comments by the previous TSE's. Our project looks great and it is very responsive 😄 I hereby approve the project. Kindly go ahead and merge to master and submit project completion form ✋ 
[OPTIONAL]
Kindly fix the image display in the readme file. The Image is not displaying

Good luck with your next projects 
_Happy coding_ 👨‍💻 👩‍💻 
Daniel Larbi Addo

**Pull request merged**